packages/client/src/index.ts

```diff
@@ -10,15 +12,23 @@ import {
 export type DecryptedMessage = {
   properties: Property[];
   sign: Signature;
-}
+};
 
 export async function registerInfo(
-  url: string,
+  requestUrl: string,
   properties: Property[],
-  // TODO: @ts-ignore should be removed, when below function is implemented.
-  // @ts-ignore
 ): Promise<Signature> {
-  // TODO: Not implemented yet.
+  const regex = /(http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/;
+  if (!regex.test(requestUrl)) {
+    throw new Error('The requestUrl is not validated one. Please check your URL.');
+  }
+  const response = await axios.post(requestUrl, { properties });
+  if (!response.status.toString().startsWith('2')) {
+    throw new Error('Something went wrong. Please make sure API path was clear or server-side function was implemented.');
+  }
+  const signature = response.data as Signature;
+
+  return signature;
```

Comment:
As you can see on the test codes, I added a URL validating condition and server response validating condition.
And if there is no problem with them, this function will get the `Signature` from the transferred URL and returns it.